tests/lint/test_pylinter.py

```diff
@@ -33,9 +32,7 @@ def test_crash_in_file(
     files = tmpdir.listdir()
     assert len(files) == 1
     assert "pylint-crash-20" in str(files[0])
-    with open(files[0], encoding="utf8") as f:
-        content = f.read()
-    assert "Failed to import module spam." in content
+    assert any(m.symbol == "fatal" for m in linter.reporter.messages)
```

Comment:
Previous test makes no sense as this test tries to lint the current file. Since the crash report includes the current file in the report whatever you put in an assertion will always end up in the crash report as well.

Since the test doesn't matter that much anyway I thought replacing was best.